core/arch/arm/plat-imx/pm/psci.c

```diff
@@ -47,9 +47,22 @@
 #include <tee/entry_std.h>
 #include <tee/entry_fast.h>
 
+int psci_features(uint32_t psci_fid)
+{
+	switch (psci_fid) {
+#ifdef CFG_BOOT_SECONDARY_REQUEST
+	case PSCI_CPU_ON:
+		return 0;
```

Comment:
Please use suitable define instead of `0`

Reply:
I think 0 is right thing here. Here's what the PSCI spec says about the return code. Since psci_cpu_on does not have any feature flags associated with it, it is a bitmask of 0. I don't think PSCI_RET_SUCCESS would be correct in this case.

![image](https://user-images.githubusercontent.com/11446053/36808726-c7b51754-1c7a-11e8-8f13-6c4d47933eca.png)

![image](https://user-images.githubusercontent.com/11446053/36808700-b76380fc-1c7a-11e8-913d-6a707ac2fe6a.png)